src/app/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from "@reduxjs/toolkit";
+import counterReducer from "../features/counter/counterSlice";
```

Comment:
why we are storing the reducer inside the feature folder can you check with rohit once